client/app/components/preprint-basics/component.js

```diff
@@ -43,14 +43,35 @@ function doiRegexExec(doi) {
 /* Does not support editing */
 export default Ember.Component.extend(BasicsValidations, {
     editMode: true,
-    uploadValid: Ember.computed.alias('nodeLocked'), // Once the node has been locked (happens in step one of upload section), users are free to navigate through form unrestricted
+    excludeContent: Ember.computed('excludeList', function(){
```

Comment:
Should this computed property depend on `options.excludeList` instead of `excludeList`?